Project-II-Calculator/calculator/src/App.js

```diff
@@ -1,23 +1,281 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 
+import CalculatorDisplay from './components/DisplayComponents/CalculatorDisplay';
+import NumberButton from './components/ButtonComponents/NumberButton';
+import ActionButton from './components/ButtonComponents/ActionButton';
+
 const App = () => {
+
+  //Initialize State
+  const [buttonSequenceState, setButtonSequenceState ] = useState([]);
+
+  //Button Click Handler
+  const buttonClickHandler = async function(value) {
```

Comment:
again, maybe start thinking of other ways to tackle this function, maybe with a switch statement or recursive functions, just play around with ideas and some great things can come from just trying new things.